mnist/mnist_vanilla_k8s.ipynb

```diff
@@ -0,0 +1,909 @@
+{
```

Comment:
<p>Delete credits; not neccessary.</p><p> </p><p>In the overview can you describe the storage solution e.g minio</p><p> </p><p>Also it looks like this depends on DockerHub for the registry. Could you call that out as well?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kubeflow/examples/pull/737/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Deleted

Reply:
<p>Added instructions on how to add docker registry and why it's used</p>